react/game/src/constants/defaultUser.js

```diff
@@ -0,0 +1,11 @@
+const DEFAULT_USER = {
```

Comment:
U can save this default user as `initialState` and avoid to use in the `mapStateToProps`